.github/ISSUE_TEMPLATE/lts-bug.yml

```diff
@@ -0,0 +1,107 @@
+name: "\U0001F41C LTS bug report"
+description: Submit a bug report to help us improve an LTS version of Iroha (iroha2-lts branch)
```

Comment:
```suggestion
description: Submit a bug report to help us improve an LTS version of Iroha ([`iroha2-lts`](https://github.com/hyperledger/iroha/tree/iroha2-lts) branch)
```

Reply:
I like that idea, sadly, Markdown is not supported for that small part of issue templates.

![](https://i.imgur.com/sFkXaBt.png)

Reply:
Then again, it seems like a little parser quirk we can resolve a bit differently, "choose a different type" part works.

Reply:
Markdown links are broken in two places out of three, issue type selection breaks, too.
GitHub preview for `yml` template recognizes the Markdown, everything else doesn't.

![](https://i.imgur.com/moLs8it.png)

Reply:
No luck here, either. I wonder why GitHub made it plain-text only and if it's worth to make an issue about later.

![](https://i.imgur.com/o7JmfW1.png)

Reply:
I'm thinking we can do this:

```yaml
name: "\U0001F41C Bug report: LTS version (iroha2-lts)"
description: Submit a bug you found in the LTS version of Iroha
```
```yaml
name: "\U0001F41E Bug report: Stable version (iroha2)"
description: Submit a bug you found in the stable version of Iroha
```